eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```diff
@@ -95,7 +96,7 @@ services:
             - '%ezpublish.image_alias.imagine.cache_resolver_decorator_relative.class%'
 
     ezpublish.image_alias.imagine.cache_resolver_decorator:
-        class: '%ezpublish.image_alias.imagine.cache_resolver_decorator_proxy.class%'
+        class: '%ezpublish.image_alias.imagine.cache_resolver.interface%'
```

Comment:
I guess we don't need param for the interface ;)

Reply:
Done in https://github.com/ezsystems/ezpublish-kernel/pull/2511/commits/8aa77c0c088dbf75a0c60f1563796b9a3c993b53.